### ISSUE TITLE
test_autosetup test_import fails

### ISSUE THREAD

**Issue by faassen** [morepath/morepath#353]
I'm not sure what is going on here, I wonder whether it's triggered by some particular combination of setuptools and stuff, though upgrading my buildout had no effect. The failure is because the `ns` module, defined in `no_mp_ns`,  somehow makes it into `morepath_packages` even though the test doesn't expect it to be. I'm curious what changed to make this happen as it seems to indicate some kind of underlying fragility.

**Comment by faassen** [comment]
I tried this with setuptools 18.0.1. I then upgraded to 19.4 but it didn't change the behavior.

**Comment by href** [comment]
I'm afraid I can't reproduce this error. I tried it with setuptools 18.x and 19.4. I also rebuilt my virtualenv and tried [the latest bootstrap.py](https://github.com/buildout/buildout/blob/9ba1e3170a474e6139c9a7eadd32a498c22cb3bb/bootstrap/bootstrap.py) to bootstrap the environment as follows:
`python bootstrap.py --setuptools-version=19.4 --buildout-version=2.5.0`.

How are you calling the tests?
May I see the stack trace?

I'm not sure this is a setuptools problem. To make sure I changed the tox.ini to always update pip and setuptools before running the tests. They all still pass on travis.
